RiotSwiftUI/Modules/Room/StaticLocationSharingViewer/View/StaticLocationSharingViewer.swift

```diff
@@ -0,0 +1,97 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct StaticLocationSharingViewer: View {
+
+    // MARK: - Properties
+    
+    // MARK: Private
+    
+    @Environment(\.theme) private var theme
+    
+    // MARK: Public
+    
+    @ObservedObject var viewModel: StaticLocationSharingViewerViewModel.Context
+    
+    // MARK: Views
+    
+    var body: some View {
```

Comment:
There's an unwanted bottom inset here which you will need to handle:
![Screenshot 2022-04-11 at 15 50 21](https://user-images.githubusercontent.com/637564/162742898-72bb1736-fdfb-4799-b705-aef751aaa121.png)